src/components/ha-spinner-delayed.ts

```diff
@@ -0,0 +1,32 @@
+import "@shoelace-style/shoelace/dist/components/animation/animation";
+import "./ha-spinner";
+import { html, LitElement } from "lit";
+import { customElement, property } from "lit/decorators";
+
+@customElement("ha-spinner-delayed")
+export class HaSpinner extends LitElement {
+  @property({ type: Number, attribute: "delay-seconds" }) public delaySeconds =
+    1;
+
+  @property() public size?: "tiny" | "small" | "medium" | "large" | number;
+
+  public render() {
+    return html`
+      <sl-animation
+        name="fadeIn"
+        .delay=${this.delaySeconds * 1000}
+        fill="both"
+        .iterations=${1}
+        play
+      >
```

Comment:
Shouldn't it just be:

```html
<ha-delay delay="1">
	<ha-spinner></ha-spinner>
</ha-delay>
```

So we dont have to duplicate the API of `ha-spinner` here and it is easier to reuse?